PCLConfig.cmake.in

```diff
@@ -645,7 +645,7 @@ endif()
 pcl_remove_duplicate_libraries(PCL_COMPONENTS PCL_LIBRARIES)
 
 # Add 3rd party libraries, as user code might include our .HPP implementations
-list(APPEND PCL_LIBRARIES ${BOOST_LIBRARIES} ${QHULL_LIBRARIES} ${OPENNI_LIBRARIES} ${OPENNI2_LIBRARIES} ${ENSENSO_LIBRARIES} ${davidSDK_LIBRARIES} ${DSSDK_LIBRARIES} ${RSSDK_LIBRARIES} ${FLANN_LIBRARIES} ${VTK_LIBRARIES})
+list(APPEND PCL_LIBRARIES ${BOOST_LIBRARIES} ${QHULL_LIBRARIES} ${OPENNI_LIBRARIES} ${OPENNI2_LIBRARIES} ${ENSENSO_LIBRARIES} ${davidSDK_LIBRARIES} ${DSSDK_LIBRARIES} ${RSSDK_LIBRARIES} FLANN::FLANN ${VTK_LIBRARIES})
```

Comment:
We will have a problem with this line if PCL is compiled without components that need Flann. In this case the `find_flann` macro is never called, Flann is never searched for, and this target is undefined. Previously this worked because `${FLANN_LIBRARIES}` would be undefined and thus treated as an empty string.

A possible solution is to undo change in this line and set `FLANN_LIBRARIES` to `FLANN::FLANN` inside `find_flann` macro.

Reply:
Used another workaround for this now ;)